src/OSPSuite.Infrastructure.Import/Core/ColumnInfo.cs

```diff
@@ -10,7 +10,7 @@ public class ColumnInfo
 
       public string Name
       {
-         get { return _name; }
+         get => _name;
```

Comment:
Resharper suggestion to shorten the code. Let's use them unless they make the code less readable